sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -361,12 +361,6 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
-  val ANSI_SQL_PARSER =
-    buildConf("spark.sql.parser.ansi.enabled")
```

Comment:
These two places also need to be updated:
https://github.com/apache/spark/blob/bacffb8810434b36fdb6fdd622a0f8c8d99ee5ab/sql/core/src/test/resources/sql-tests/inputs/ansi/interval.sql#L2
https://github.com/apache/spark/blob/261e113449cf1ae84feb073caff01cc27cb5d10f/sql/core/src/test/resources/sql-tests/inputs/pgSQL/text.sql#L49

Reply:
@wangyum Thanks